internal/server/handler_v1.go

```diff
@@ -183,38 +183,78 @@ func (s *Server) VariableInfo(
 	return info.VariableInfo(ctx, in, s.store)
 }
 
+var storeBulkVariableInfo = info.BulkVariableInfo
+
+var remoteBulkVariableInfoResponse = func(
+	s *Server,
+	remoteReq *pbv1.BulkVariableInfoRequest,
+) (*pbv1.BulkVariableInfoResponse, error) {
+	remoteResp := &pbv1.BulkVariableInfoResponse{}
+	return remoteResp, util.FetchRemote(
+		s.metadata,
+		s.httpClient,
+		"/v1/bulk/info/variable",
+		remoteReq,
+		remoteResp,
+	)
+}
+
 // BulkVariableInfo implements API for mixer.BulkVariableInfo.
 func (s *Server) BulkVariableInfo(
 	ctx context.Context, in *pbv1.BulkVariableInfoRequest,
 ) (*pbv1.BulkVariableInfoResponse, error) {
-	localResp, err := info.BulkVariableInfo(ctx, in, s.store)
-	if err != nil {
+
+	errGroup, errCtx := errgroup.WithContext(ctx)
+
+	storeResponseChan := make(chan *pbv1.BulkVariableInfoResponse, 1)
+	remoteMixerResponseChan := make(chan *pbv1.BulkVariableInfoResponse, 1)
+
+	errGroup.Go(func() error {
+		storeResponse, err := storeBulkVariableInfo(errCtx, in, s.store)
+		if err != nil {
+			return err
+		}
+		storeResponseChan <- storeResponse
+		return nil
+	})
+
+	if s.metadata.RemoteMixerDomain != "" {
+		errGroup.Go(func() error {
+			remoteMixerResponse, err := remoteBulkVariableInfoResponse(s, in)
```

Comment:
nit: remoteResponse to match storeResponse (localResponse)

Reply:
Done.